### PR TITLE
kernel: Fix that the help is not printed twice

### LINKED ISSUE #433: build-kernel: prints help twice if run with no args
If you run the kernel `build-kernel.sh` command with no arguments, it prints the help out twice. It just bugs me every time I do that when I'm trying to remember what the subcommands actually are :-)

### PR DESCRIPTION
**Pull request by GabyCT**
While running the build-kernel.sh script with no arguments, the help is
printed twice. This PR will fix that.

Fixes #433

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by grahamwhaley** [comment]
/test-ubuntu
Not that I think any CI will test this..

**Comment by jcvenegas** [comment]
```
08:16:37 kernel/build-kernel.sh
08:16:37 Kernel directory has changes check kernel/kata_config_version changed
08:16:37 ERROR: Please bump version in kernel/kata_config_version
```

File: kernel/kata_config_version [hunk 1]
```diff
@@ -1 +1 @@
-55
+56
```
Comment (amshinde) [inline kernel/kata_config_version]: This sort of change should not require a bump in the config version. We should fix this in a future PR. We should enforce a version bump only if any of the kernel configuration changes imho.

**Pull request merged**